e3nn_jax/_src/s2grid.py

```diff
@@ -163,17 +169,20 @@ def from_s2grid(x: jnp.ndarray, lmax: int, normalization="component", lmax_in=No
         lmax (int): maximum degree of the spherical tensor
         normalization ({'norm', 'component', 'integral'}): normalization of the spherical tensor
         lmax_in (int, optional): maximum degree of the input signal, only used for normalization purposes
+        quadrature (str): "soft" or "gausslegendre"
+        p_val (int): one argument for determining the parity of the resulting spherical tensors
+        p_arg (int): the other argument for determining spherical tensor parity
```

Comment:
It's technically more about the input parity (`val` and `arg` stand for the value and argument of the input signal, which then translate to the parity of the output... as we derived together last time)

```suggestion
        p_val (int): the parity of the value of the input signal 
        p_arg (int): the parity of the argument of the input signal
```